auth-web/src/components/auth/manage-business/ManageBusinessDialog.vue

```diff
@@ -392,7 +392,9 @@ export default defineComponent({
       nameOption.value = false
       requestAuthBusinessOption.value = false
       requestAuthRegistryOption.value = false
-      addBusinessForm.value.resetValidation()
+      if (!props.isStaffOrSbcStaff) {
```

Comment:
Is this needed?

Reply:
Yes, this is needed for when resetForm is called from SearchBusinesnameRequest... Without this, there is a console error when resetValidation is called and the business search bar doesn't get cleared

Reply:
Can you write that in a comment above it?